core-blocks/file/edit.js

```diff
@@ -96,10 +79,7 @@ class FileEdit extends Component {
 				fileName: media.title,
 				textLinkHref: media.url,
 				id: media.id,
-			} );
-			this.setState( {
-				attachmentPage: media.link,
-			} );
+			}, this.props.media.get /* refetch attachment page url */ );
```

Comment:
Does `setAttributes` accept a second argument? When is `this.props.media.get` called?

Reply:
Totally assumed `setAttributes` took the same arguments as `setState`! I meant it as a callback. Thank you.